avatars/avatars.js

```diff
@@ -2577,27 +2479,47 @@ class Avatar {
             })();
             if (!isPosition) {
               if (useAnimation) {
-                const src2 = useAnimation.interpolants[k];
+                const idleAnimation = _getIdleAnimation('walk');
+                
+                let src2 = useAnimation.interpolants[k];
+                
+                if(this.move && k.includes("Leg") || this.move && k.includes("Foot") || this.move && k.includes("Toe") || this.move  && k.includes("Hips"))
```

Comment:
This is a weird duplicated condition.

Reply:
It could be made simpler with better layering of the statements.

Reply:
will update this to cleaner code

Reply:
Updated